diff -dur mmueller2012-pipelight-b7b5e5471d52.orig/bin/pipelight-plugin.in mmueller2012-pipelight-b7b5e5471d52/bin/pipelight-plugin.in
--- mmueller2012-pipelight-b7b5e5471d52.orig/bin/pipelight-plugin.in	2014-12-10 05:56:45.000000000 +0100
+++ mmueller2012-pipelight-b7b5e5471d52/bin/pipelight-plugin.in	2015-01-27 22:10:38.136330589 +0100
@@ -93,7 +93,7 @@
 	PLUGIN_PATH=$(realpath "$PLUGIN_PATH")
 
 	# Global installation
-	if [ $(/usr/bin/id -u) -eq 0 ]; then
+	if [ $(/bin/id -u) -eq 0 ]; then
 		INSTALLDIR="$PLUGIN_PATH"
 
 	# Use environment variable (only if it doesn't point to the global directory)
@@ -421,7 +421,7 @@
 	local retval=0
 
 	# Don't run this as root
-	if [ $(/usr/bin/id -u) -eq 0 ]; then
+	if [ $(/bin/id -u) -eq 0 ]; then
 		echo "ERROR: You should not run this as root!" >&2
 		return 1
 	fi
@@ -545,6 +545,10 @@
 			return 1
 		fi
 	done
+	
+	if [ "$INSTALLDIR" = "$PLUGIN_PATH" ] ; then
+		/usr/sbin/update-browser-plugins || :
+	fi
 
 	return 0
 }
@@ -569,6 +573,10 @@
 
 	done
 
+	if [ "$INSTALLDIR" = "$PLUGIN_PATH" ] ; then
+		/usr/sbin/update-browser-plugins || :
+	fi
+
 	return 0
 }
 
diff -dur mmueller2012-pipelight-b7b5e5471d52.orig/share/install-dependency mmueller2012-pipelight-b7b5e5471d52/share/install-dependency
--- mmueller2012-pipelight-b7b5e5471d52.orig/share/install-dependency	2014-12-10 05:56:45.000000000 +0100
+++ mmueller2012-pipelight-b7b5e5471d52/share/install-dependency	2015-01-27 22:10:18.286240248 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
 usage()
 {
diff -dur mmueller2012-pipelight-b7b5e5471d52.orig/share/scripts/configure-flash.in mmueller2012-pipelight-b7b5e5471d52/share/scripts/configure-flash.in
--- mmueller2012-pipelight-b7b5e5471d52.orig/share/scripts/configure-flash.in	2014-12-10 05:56:45.000000000 +0100
+++ mmueller2012-pipelight-b7b5e5471d52/share/scripts/configure-flash.in	2015-01-27 22:10:18.286240248 +0100
@@ -2,7 +2,7 @@
 PIPELIGHT_SHARE_PATH="@@PIPELIGHT_SHARE_PATH@@"
 
 # Don't run this as root
-if [ $(/usr/bin/id -u) -eq 0 ]; then
+if [ $(/bin/id -u) -eq 0 ]; then
 	echo "ERROR: You should not run this as root!" >&2
 	exit 1
 fi
diff -dur mmueller2012-pipelight-b7b5e5471d52.orig/share/scripts/configure-silverlight.in mmueller2012-pipelight-b7b5e5471d52/share/scripts/configure-silverlight.in
--- mmueller2012-pipelight-b7b5e5471d52.orig/share/scripts/configure-silverlight.in	2014-12-10 05:56:45.000000000 +0100
+++ mmueller2012-pipelight-b7b5e5471d52/share/scripts/configure-silverlight.in	2015-01-27 22:10:18.289573597 +0100
@@ -2,7 +2,7 @@
 PIPELIGHT_SHARE_PATH="@@PIPELIGHT_SHARE_PATH@@"
 
 # Don't run this as root
-if [ $(/usr/bin/id -u) -eq 0 ]; then
+if [ $(/bin/id -u) -eq 0 ]; then
 	echo "ERROR: You should not run this as root!" >&2
 	exit 1
 fi
@@ -68,4 +68,4 @@
 # Cleanup
 rm "$tmpfile"
 
-exit 0
\ No newline at end of file
+exit 0
